Scifi short story (#5)
sciencectn/sciencectn.github.io: _posts/2023-05-19-story.md

---
layout: post
title: Dissent Module - A short story
date: 2024-05-19
permalink: story
---

<!--  SETIbro -->

`December 24th`

`Near Kennedy Space Center, Florida`

 They gathered around a table lit by a lone pair of overhead lights in an otherwise dark sea of cubicles and desk chairs. On the table were to-go boxes of Chinese food, a lit Hanukkah menorah, and a portable speaker playing Bollywood songs. Some wore ugly holiday sweaters, some had turbans, one had a `hello world` computer programmer shirt. All had their NASA badge lanyards around their necks. 
 

>  The woman wearing the blue flight suit was the loudest. Her mission patch on her shoulder showed "LEO Lodge" encircling a cartoon of a beaver building a space station. Her chest patch showed *Eva Resnik*. She chatted with the Sikh sitting next to her wearing a turban with galaxies on it. 

Reviewer: ```suggestion
 The woman wearing the blue flight suit was the loudest. The mission patch on her shoulder showed "LEO Lodge" encircling a cartoon of a beaver building a space station. Her chest patch showed *Eva Resnik*. She chatted with the Sikh person sitting next to her wearing a turban with galaxies on it. 
```

Eva had sarcasm in her tone. "Talia, what if we just found aliens!"

"This thing finds aliens a few times a year."

<!-- Clocks -->

Eva shuffled out of bed and opened the shades. It was early morning, a soft blue glow emerging from the east. She checked the digital clock on the wall. Then blinked and checked it again. 

*1:30am*. 

She looked out the window again at the blue glow on the horizon. The streets were quiet. She checked her phone. No cell service. Its clock also showed *1:30am*. She checked the wifi. Infinite loading spinner. 

"The fuck?"

She got in her car and furrowed her brow at the cacophony of warning popups on the center touch screen. 

`Lost connection to NULL.`

`We're sorry, {MSG_TEXT_HERE}`

She tapped all the buttons on the screen, which didn't respond. "Piece of shit."

She grabbed her bike, rode it to Talia's apartment, dashed up the stairs, then knocked on the door. "Talia! It's Eva!"

After a few minutes, Talia emerged: groggy, squinting, her hair disheveled, wearing *Piranha Plant* pajamas. "Oh, hey."

"Your car. Does it start?"

Talia blinked. "What the hell? I've told you many times before: it works fine. I don't need that connected shit."

> Talia's eyes widened when she realized Eva was serious. "Wait, what's going on?"

Reviewer: ```suggestion
Talia blinked. "What the hell? I've told you many times before: it works fine. I don't need that connected shit." Her eyes widened when she realized Eva was serious. "What's going on?"
```

Talia clicked around the analysis windows and noticed something. "This FM radio thing is odd. The phased array is showing two radio signals. There's the signal down on earth then another signal from the sky. I'm going to compare the two."

She played back the song from the ground station: _Man in Black_ by Johnny Cash. Then she played the other interfering signal from above. It sounded the same. 

"It sounds fine," Eva said. "Why were we getting static on the drive over here?"

Talia superimposed the two and zoomed in. "Look at that."

One line showing a sine-looking wave. The other showed the exact same wave...upside down. 

Eva said, "Destructive interference. I remember that from HAM radio training. That's when we screw up and kill a signal by transmitting its inverse." 

Eva looked outside again. "I want to try something."

She rushed outside and looked at the GPS readout on her phone. Surprisingly, it got a fix and reported high confidence in its location, showing them in the middle of the Indian Ocean. Just as Talia followed her outside, Eva rushed back inside. 

"What are you looking for?" Talia asked.

> She darted over to grab a globe off someone's desk, then marked the location with a sharpie.

Reviewer: ```suggestion
She darted over to grab a globe from someone's desk, then marked the location with a sharpie.
```

Eva typed 

`hello I'm Eva we come in peace.`

"Really? Alright." She hit enter. Immediately, some gibberish text showed up. "Wow, that was fast. Here's the response."

Eva puzzled over it. "Huh. Can you make anything of it? Is it doing opposites again?"

She tried a few commands. "Yep. Every bit of information is flipped. Wait a minute, there's more."

New text showed up. 

`WAY afar afar afar afar afar`

The text repeated itself. 

They looked incredulously at each other. "The fuck?" Eva said. 

Eva typed a reply. 

`I don't understand. Could you explain?`

The response came back. 

`Eva shall make physical contact with the information forager. Forager ceases electromagnetic communication.`

They both looked at each other. 

"Information forager? What on earth is that?" Eva asked.

"And you have to make 'physical contact' with it?"

"Like, I have to physically go up there and touch it?"

<!-- Aftermath -->

Days later, Eva scrolled through the news headlines. 

_Global Crisis Unfolds as Mystery Satellite Disrupts Earth's Telecom: What We Know So Far_

_World Leaders Demand Answers After Space-Borne Attack Disables Global Communications_

> _How this Quick Thinking First Responder Used Morse Code to Reach Crash Victims_

Reviewer: Reading news headlines and social media posts as a way to give exposition is a cliche -- maybe try the Quick Thinking First Responder one as a single example of what Eva happens to have scrolled to at a particular moment in time, within some other scene? I like that one since it gives us information about the culture within the story's world (e.g., clickbait headlines are still a thing) and hints at the damage and harm that the event has caused.

He didn't seem interested in smalltalk. 

"I apologize that Bill couldn't be here in person to deliver the news himself. I'll cut to the chase. What I'm about to tell you is classified."

She nodded. 

"As you probably know, tensions with the EAC were already high after Flight 766. Now it's worse. Two of their civilian aircraft crashed and we still don't know the status of the victims. We have high confidence it was because GPS spoofing from the electronic warfare attack caused their autopilot to malfunction. They're blaming us for the attack."

He continued. 

"And now for the classified part, which is why I brought you here. We have a pretty good idea of the EAC's and other adversaries' capabilities. We know our capabilities. Whatever did this is beyond *anyone's* capabilities."

"In what way?" she asked. 

"The electronic warfare attack, while impressive, is within the realm of possibility. But it's not _how_ they did the attack we're interested in. It's _where_ the attack came from. It essentially came out of nowhere and shows no resemblance to any known satellite's orbit, including those of our adversaries."

"And what did the EAC say about that?"

> "Our backchannels in the EAC said cooler heads have not prevailed. Their regime is under intense pressure for legitimacy and they might lash out to score some political points. They're moving destroyers with anti-satellite missiles into offensive positions."

Reviewer: I like "cooler heads have not prevailed" as indirect, and wry, military-speak. Nice touch in this dialog

He continued. 

> "That's where you come in. That _Sunfly_ rocket to LEO Lodge? It's getting repurposed to visit the unknown object. You're going to rendezvous with it and get whatever information you can. It asked for you so that's who we're sending."

Reviewer: "It asked for you so that's who we're sending" feels a bit like a stretch to me in the plot. Would the government have tried to make contact with some other operative claiming to be Eva, either from earth or by sending up the rocket (or another vehicle)? If so, would the unknown object detect the deceit?

Author: Good point, I'll address this.

Eva felt uneasy. She wasn't sure whether to be excited or scared. 

"Alright. Great! Who's on my crew?"

"You will be accompanied by extra hydrazine."

"So, more fuel."

"The object is in a higher orbit than LEO Lodge and the current configuration won't get that far. Replacing some crew with fuel should get us where we need. Any questions?"

"I want Talia on comms. She knows this thing better than anyone else."

"Talia Wescoff? I think we can arrange that."

<!-- Launch -->

> Eva looked at the black sky through the crew capsule. 

Reviewer: Are you planning to make these visible sub-headers between sections, and/or are you planning to fill out these scene transitions more? I like that they err on the side of stopping once they've advanced the story as far as you need them to, rather than being padded out, but the first few are abrupt. Maybe try extending them a bit based on the emotion or character motivation you want each scene to present. Eva meets with Eckert, expecting him to punish her but instead finds out she's been tagged for a solo space mission. How is she feeling when she exits this conversation -- afraid, excited, unreal? Is there a tangible thing she would do on her way out of this scene -- leaving the meeting room, walking outside, arriving back at her home -- that would show some of what's going on inside her?

Author: Originally I had them as subheaders but it broke the flow. They're just for my own uses.

"Initiate roll program." 

The sliver of moonlight streaming through the window crept over the instrument panels as the rocket rolled into its climb position. 

"T minus 10 seconds to MECO."

"Mark. Main engine cut off."

The rocket went silent. Eva checked the radar. Talia spoke over the radio. "20 minutes to rendezvous. Eva you got anything yet?"

"Nothing yet."

Some minutes later, a radar ping grabbed her attention. 

"Wait, I've got it. Very clear radar return. Making course corrections now."

The thrusters made hundreds of small _hiss_ noises as they perfectly aligned the ship to intercept the unknown craft. 

Talia's voice crackled in her earpiece. "Eva, CAPCOM. 10 seconds until orbital insertion burn."

The rocket on the crew capsule fired. Eva was pressed into her seat. She watched the distance to the unknown object decrease. 

_2km_

_1km_

_400m_

The craft went silent. Eva squinted out the window. The unknown object was a silver dot. "I have a visual. Moving to rendezvous."

She used the more gentle rendezvous thrusters to approach the object. The relative distance numbers dropped more slowly this time.

_300m_

_200m_

_100m_

> The silver dot enlarged in the capsule's window and then took shape. The object looked cylindrical with curved surfaces with a hollow interior, like an turbofan engine nacelle on a jumbo jet or the nozzle of a hairdryer. Except, there was nothing in the inside, as if it was a nacelle without an engine. The empty hole in the interior with the smooth circular lip almost looked like the mouth of a lamprey. It had the diameter of an average passenger jet and the length of about two sedans front-to-back. 

Reviewer: I like the juxtaposition of a jet engine nacelle and the nozzle of a hairdryer as comparisons coming to Eva's mind (assuming the narration is following her POV), especially the absurdity of the hairdryer as an impression.

Talia and the rest of mission control stared at the video feeds in silence. The craft came closer. _10 meters_

Eva spoke into her mouthpiece, "I hope you're getting this on the video feed. I'm going to put the capsule in parking mode and go up to it on a tether."

She tapped through a few screens on the console. *Auto-park*. This would keep her capsule stationary relative to the unknown object. 

"Moving up to it now."

She donned her spacesuit and attached a tether cable. The unknown object seemed like *up* to her as the bright blue of the earth lay below. She pushed off the capsule and floated "upward" toward it, using the MMU thrusters on her spacesuit to slow down just beside it. 

Talia squinted at the shaky video feed. "Eva, CAPCOM. Can you describe what you see?"

"The outside has these metallic looking hexagonal fish scales with dark little creases between them. The whole surface seems uniform. I don't see any thrusters or antennas. It doesn't make sense. Talia, you getting anything on the sensors?"

> "Still nothing. It's quiet, as it promised."

Reviewer: Your dialog, and the balance between dialog and narration, are overall very good! There are some exchanges like this one where I'd experiment with paring them down a little more -- I sometimes want to find out more about how the characters are feeling than what they're thinking or seeing, and/or to see more contrast in how the characters are reacting, and that might help.

```suggestion
Talia squinted at the shaky video feed. "Eva, CAPCOM. Can you describe what you see?" [This is good: Talia is working to see and understand, but can't make it out. I expect she feels uncertain, off balance, anxious]

"It doesn't make sense. [Lead with this: Eva is puzzled but keeping her cool, trying to figure it out] The outside has these metallic looking hexagonal fish scales with dark little creases between them. [Remove "The whole surface seems uniform" since it contradicts the fish scales and creases] I don't see any thrusters or antennas. Talia, you getting anything on the sensors?"

"It's quiet." [What if Talia's response here is just this? It sounds more clipped and more pensive, in contrast to Eva's thinking out loud, and keeps Talia's focus on what she doesn't know instead of what she does know, e.g. that the object promised to be quiet]
```

"Received. Here goes nothing."

She slowly reached out her sausage-like gloved finger, the pressurized suit keeping her fingers inflated. She touched the surface then withdrew her hand immediately, waiting for a reaction. Nothing. She ran her fingers along the surface. 

"It feels smooth with some undulating cracks. I'm not getting any sign of life." 

Talia replied. "It said 'physical contact'. Maybe that's sound waves. Can you listen to it?"

"I'm going to put my helmet up to it."

She held on to the curved lip at the edge of the hole, one hand on the dark inside, the other on the sunny outside. She rotated her whole body forward until her faceplate made a _thud_ against the object's surface. 

"Hello. This is Eva. Remember? We communicated a while ago. I'm here."

She waited. 

"I'm not getting anything," she said over the radio. 

Suddenly, she heard high pitched noises, almost like scratching. It sounded like a dentist's cleaning instrument moving around inside her helmet.

"You hearing that?" Eva said. 

<!-- Forager -->

"Yes, it sound scratchy." 

She heard a voice. A faint voice. It sounded like a girl's voice. 

> "way afar afar afar hello hello afar afar hello hello"

Reviewer: Nice

It increased in volume. 

"Yes? I hear you."

"Eva makes..." it said. She couldn't hear the rest.

"I can't hear. It's too quiet." 

Louder this time. "Eva makes physical contact," it said.

Eva recoiled, pulling her helmet away so it wouldn't hear.

"Ok, I think it's talking. Also it sounds like...me. When I was a kid."

She made contact with her helmet again. "What's your name?"

> "The name's Bond, James Bond."

Reviewer: Aliens communicating in pop culture references is also a cliche. The voice sounds like Eva as a little girl -- does the object have access to Eva's thoughts and memories? If so, what if it replied with the name of a childhood imaginary friend or toy, or an old nickname? Or an in-joke that no one else would know? In any case, what is the object trying to do by replying with the James Bond line? Confuse her, amuse her, demonstrate awareness of her culture, something else?

It was quiet, as if thinking. Eva tried again. "You are some kind of spacecraft. I'm touching you with my helmet right now. What do I call you?"

"Eva is making contact with an Interstellar Information Forager."

"Good! Ok, Forager. I'm calling you Forager. What are you doing here?"

"Assigning pronouns and names to the Forager probe runs the risk of unnecessary anthropomorphizing and imbues it with capabilities and intentions that may not exist". 

She grimaced, hearing her own childlike voice.

"Yeah, well, we gotta start somewhere. So why did you choose to sound like a younger version of me if you don't want people anthropomorphizing?"

"It felt right."

"Felt? You have feelings?"

"This is the closest concept in your language. The Forager probe is equipped with reinforcement modules. Each one monitors its sensors and either reinforces or punishes the behavior modules according to mission goals. Because there are competing goals, there is a system of arbitration that assigns priorities to each goal. I am not permitted to know the parameters of the arbitration. All I know is some behaviors are reinforced."

"Ok. So you have goals and you feel happy when you achieve the goals."

> "This is not an entirely incorrect metaphor and will suffice given the time constraints."

Reviewer: Forager referring to "time constraints" is unnerving, which I like, though I'd like to see Eva (or Talia or someone else back at mission control) notice it and ask what those constraints are. (Or, if Forager is referring to the US's political situation, be direct about that)

Hundreds of kilometers downrange of the destroyer, the submarine _HMS Gloucester_ was ready. 

A satellite spotted the launch. It blinked the information to the waiting sub. White sea foam glowed and bubbled. A missile burst out of the water and streaked away. Its tip glowed white hot. It plowed through the air at over mach 10. It tracked the anti-satellite missile. The two points of light converged together in the night sky. The threat ended in an explosion. 

Eva scrambled back in to the capsule and went to close the hatch. 

"Eva, CAPCOM. Missile down. Looks like it got intercepted. I think one of our friends bought you some time."

"CAPCOM, Eva. I'm going back out."

"You don't have much time. They're going to launch again. The next one may hit its target."

She kicked off the capsule and used her spacesuit's MMU to maneuver back into position. She hit Forager a bit too fast and swung around, holding on to the lip to stay put. She pressed her helmet back to its surface.

"Hey, I'm here. You were saying something about autoimmune? Cleansing?"

"We do not have much time. Your faction's rival is going to strike again."

"I know, it's ok. Keep going."

> "The Body's immune function regularly repairs or purges defective modules. However, in this case, I collected evidence that it was acting in a defective manner. Autoimmune, as you call it. I showed that if its collection methodology were applied universally it would lead to our destruction." 

Reviewer: Intriguing

Eva had an idea.  

"It's going to hit us. It's inevitable. There is no electromagnetic transmission you can do that would stop it."

"This is not accurate. If the missile is tracking us via radar, I could interrupt its sensing abilities."

"They already thought of that. It will lock on to your jamming or countermeasure or whatever you plan to do. It's inevitable."

"Not true. I could also send a extremely high amplitude signal focused at the reception sensors of the missile. This would destroy sensitive circuitry."

Eva smiled. "This isn't possible. No one's made countermeasures like this."

"This is within my capabilities. Observe."

Eva noticed a warning on her suit from the interference. `Downlink lost. Attempting to reestablish...`

Eva clung to Forager and closed her eyes. She grimaced, expected impact. She opened one eye and looked down. She saw a moving point of light below them. The light blinked toward them and streaked by. 

"Its guidance capabilities are neutralized. It has missed."

She watched it fly off into space. 

"Ha ha!" she yelled. "You goddamn beautiful...uh...space tube!"

> Her vision blurred as the tears clung to her eyes in the zero gravity. 

Reviewer: This emotional pay-off isn't quite earned for me -- I'd like more evidence from this scene that Eva is scared and on edge. (A few lines ago she was smiling as she thought of her reverse psychology gambit)
